.github/ISSUE_TEMPLATE/v8_breaking_change.md

```diff
@@ -2,7 +2,7 @@
 name: 8.0 Breaking change
 about: Breaking changes from 7.x -> 8.0
 title: "[Breaking change]"
-labels: Team:Elasticsearch UI, Feature:Upgrade Assistant
+labels: Team:Elasticsearch UI, Feature:Upgrade Assistant, Breaking Change
```

Comment:
I added the "Breaking Change" label to differentiate these from the other issues associated with the Upgrade Assistant. This would be a new label, I'm not tied to the name. I also considered making it `Feature:Upgrade Assistant - Breaking Change`. Any preference or thoughts on a different way to classify the two different types of issues?

Reply:
Love it! What do you think of using the "Breaking change" label as a dimension for categorizing it by how it will be surfaced? We can create three labels:

`Breaking Change`
`Breaking Change:Deprecation Log`
`Breaking Change:Deprecation API`

By default, we can apply `Breaking Change` via this template. As these issues are created we'll triage them by swapping this label out for one of the others. Or do you think this overloads the label too much / adds too much complexity?

Reply:
Sounds good to me!